hazelcast/src/main/java/com/hazelcast/monitor/impl/HotRestartStateImpl.java

```diff
@@ -56,12 +69,14 @@ public void fromJson(JsonObject json) {
         final String jsonBackupTaskState = getString(json, "backupTaskState", null);
         final int jsonBackupTaskCompleted = JsonUtil.getInt(json, "backupTaskCompleted", 0);
         final int jsonBackupTaskTotal = JsonUtil.getInt(json, "backupTaskTotal", 0);
+        final boolean jsonIsHotBackupEnabled = JsonUtil.getBoolean(json, "isHotBackupEnabled");
```

Comment:
We need to use overloaded version of `JsonUtil#getBoolean` which takes a default value for the cases where "isHotBackupEnabled" is not present in JSON. This is needed to provide backward compatibility between newer versions of mancenter and older versions of hz.